test/RadworksDripsGovernance.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.20;
+
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ProposalTest} from "test/helpers/ProposalTest.sol";
+
+abstract contract RadworksDripsGovernance is ProposalTest {
+  function setUp() public virtual override(ProposalTest) {
+    ProposalTest.setUp();
+    _upgradeToBravoGovernor();
+  }
+
+  function _proposePassAndExecuteDripsProposal(string memory _description, bytes memory _callData)
```

Comment:
Is this method necessary? It seems like a little less generic version of [`_queueAndVoteAndExecuteProposalWithBravoGovernor`](https://github.com/ScopeLift/radworks-governor-upgrade/blob/3d870db42fd9e4017c595d0d783db6741a8478aa/test/helpers/ProposalTest.sol#L279)

Reply:
Done: 0d211b97e6805c7d616ed2c0b80d118c3c64809c